src/i18n/en/en.js

```diff
@@ -0,0 +1,138 @@
+// use lowercase keys for all none 1:1 translations e.g. transaction_plural => TXs
+export const en = {
+  translation: {
```

Comment:
at my previous job, when writing translations, we went by a strictly key => value setup where rather than using english terms for keys, we'd use nested keys that were specific to each area...For example:

```
translation: {
    GENERAL: {
      height: "Height",
      mined_by: "Mined By"
      ...
```

I went with English keys this time because it would be a little easier for me to put into this file (I wouldn't have to come up with every key for every file), but if we'd like to switch to the more formal setup, we can